src/main/java/no/nav/veilarbperson/client/pdl/domain/VergemaalEllerFullmaktOmfangType.java

```diff
@@ -9,14 +10,20 @@
     PERSONLIGE_INTERESSER("personligeInteresser"),
     OEKONOMISKE_INTERESSER("oekonomiskeInteresser");
 
-     private String type;
+    private String type;
```

Comment:
Er dette for serialisering? Tror kanskje dette kunne vært løst enklere med å bruke `@JsonValue` på `type`-feltet (basert på slik det var før denne endringen)

Reply:
Malapropos, men kunne strengt tatt refaktorert bort bruken av `BY_NAME` og heller brukt en stream på `values()`. Fin use case for å returnere `Optional` også IMHO.